blog/_posts/2024-06-25-history-ele-raid-buffs.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: The Historical Precedent for an Elemental Raid Buff
+#            dd/mm/yyyy
+last_update: 25/06/2024
+excerpt: Elemental Shaman's place as a raid buff provider.
+author: Lucenia
+toc: true
+big_article: true
+---
+
+> Lucenia is a 15-year veteran of Elemental Shaman currently Cutting-Edge raiding in [\<Noted\>](https://raider.io/guilds/us/akama/Noted) on US-Akama. She streams mythic raiding, mythic+, and PTR/beta testing content at https://twitch.tv/Lucenia.
+She also produces sims, guides, and other content for Elemental Shaman in Cataclysm Classic and Season of Discovery.
+
+Greetings fellow Shamans,
+
+It's no question that one of the most controversial - and often memed about - aspects of Elemental Shaman in retail World of Warcraft is its lack of a raid buff or unique raid utility.
+With the recent addition of a Hunter's Mark](https://wowpedia.fandom.com/wiki/Hunter's_Mark) raid buff and the potential for Restoration Shamans to provide powerful utility with their Totemic hero tree in combination with [Ancestral Vigor](https://wowpedia.fandom.com/wiki/Ancestral_Vigor) Elemental now stands as the lone specialization that lacks a raid buff or unique utility.
```

Comment:
Could these two links also be wowhead links instead? This way they would get tooltips too.